bin/p2-dsctl/main.go

```diff
@@ -55,11 +59,19 @@ var (
 
 	cmdUpdate       = kingpin.Command(CmdUpdate, "Update a daemon set.")
 	updateID        = cmdUpdate.Arg("id", "The uuid for the daemon set").Required().String()
-	updateAZ        = cmdUpdate.Flag("az", "The availability zone of the pod cluster").String()
+	updateSelector  = cmdUpdate.Flag("selector", "The node selector, uses the same syntax as the test-selector command").Default(SelectorNotSpecified).String()
```

Comment:
The default value is not required. Kingpin string arguments are always pointers to strings (`*string`). This means you can test the pointer if it's equal to `nil` to decide if the user has passed it or not.